additonal steps to create and activate a conda enviorment in Snakemake Tutorial (#3032)
galaxyproject/training-material: topics/data-science/tutorials/snakemake/tutorial.md

> ### {% icon hands_on %} Hands-on: Install Snakemake
> We're about to start doing things really with snakemake, so, it's time to install it.
> 1. [Install Miniconda](https://docs.conda.io/en/latest/miniconda.html)
> 2. Once you've done that and activated conda (you should see `(base)` in your terminal prompt),

> > 3. lets create a conda enviorment for snakemake by using the command 'conda create -n snakemake', and activate by using: 'conda activate snakemake'.

Reviewer: ```suggestion
> 3. Let's create a conda enviorment for snakemake and activate it with the following commands:
>    ```
>    conda create -n snakemake
>    conda activate snakemake
>    ```
```

> > 4. run `conda install snakemake` to install snakemake.

Reviewer: ```suggestion
> 4. Install snakemake with:
>    ```
>    conda install snakemake
>    ```
```
